lib/iris/tests/unit/fileformats/um/optimal_array_structuring/test_optimal_array_structure.py

```diff
@@ -115,7 +121,13 @@ def test_2d_with_element_values(self):
         shape, primaries, elems_and_dims = optimal_array_structure(
             elements, elements_values
         )
-        self.assertEqual(shape, (2, 3,))
+        self.assertEqual(
+            shape,
+            (
+                2,
+                3,
+            ),
+        )
```

Comment:
This formatting looks really horrible, is less readable and seems gratuitous. Is there a configuration we can use to make Black less zealous?